dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -345,6 +345,7 @@
                             Padding="{TemplateBinding Padding}"
                             IsTabStop="False"
                             Grid.ColumnSpan="4"
+                            VerticalContentAlignment="Center"
```

Comment:
Shouldn't it be a template binding that respect `VerticalContentAlignment`, like at line 140?

Reply:
Very good point, fixed that now (and also bound the horizontal content alignment in case someone needs it at some point).